### DISCUSSION TITLE
bot.corpus（语言库）功能如何添加多个语句？

### DISCUSSION THREAD

**Discussion by Xhaokan** [Tsuk1ko/CQ-picfinder-robot#286] [问题求助] [answered]
如果我要添加多项内容应该用什么形式填写呢？
"corpus": [{
  "regexp": "^射射竹竹$",
  "reply": "不可以涩涩o",
  "scene": "a"
}]

**Comment by Tsuk1ko** [comment, marked as answer]
`corpus` 是一个数组
